zeebe/snapshot/src/main/java/io/camunda/zeebe/snapshots/PersistedSnapshot.java

```diff
@@ -100,6 +105,26 @@ public interface PersistedSnapshot {
    */
   ActorFuture<SnapshotReservation> reserve();
 
+  default boolean isBootstrap() {
+    return getMetadata() != null && getMetadata().isBootstrap();
+  }
+
   @VisibleForTesting
   boolean isReserved();
+
+  default Map<String, Path> files() {
+    final var map = new HashMap<String, Path>();
+    try (final var stream = Files.list(getPath())) {
+      stream.forEach(
+          file -> {
+            final var fileName = file.getFileName().toString();
+            map.put(fileName, file);
+          });
+
+    } catch (final Exception e) {
```

Comment:
:question: Is this `Exception` or `IOException`? If we can specify, even better.